Packs/ServiceNow/Scripts/ServiceNowCreateIncident/ServiceNowCreateIncident.py

```diff
@@ -65,7 +66,8 @@ def get_user_id(user_name):
 def get_group(query):
     group_args = {
         'table_name': 'sys_user_group',
-        'query': query
+        'query': query,
+        'using': demisto.args().get('using')
```

Comment:
Either use the global variable `using` or propagate it as a function parameter.
```suggestion
        'using': demisto.args().get('using'),
```